incubator/jenkins/templates/home-pvc.yaml

```diff
@@ -10,7 +10,12 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
   annotations:
+{{- if not (empty .Values.Persistence.StorageClass) }}
+    volume.beta.kubernetes.io/storage-class:  {{ .Values.Persistence.StorageClass | quote }}
     volume.alpha.kubernetes.io/storage-class: {{ .Values.Persistence.StorageClass | quote }}
```

Comment:
I think `volume.alpha.kubernetes.io/storage-class` has been deprecated - but cannot find where I read that